### PR TITLE
Update JSON file (Mega Sumo)

### PR DESCRIPTION
**Pull request by Wesdial**
- Added the correct coordinates
- Made First platform break
- Added a proper world border

### CODE REVIEW

**Comment by Owehttamy** [comment]
I suggest adding `execute at @r run world border damage buffer 0` to the world border broadcast commands, as it makes players take damage right when the border passes them instead of when they're 6+ blocks away from the border.

**Comment by Wesdial** [comment]
Done

File: Blitz/Mega Sumo/map.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
 				"commands": []
 			},
 			{
-				"message": "&cThe first platform will break in 5 seconds!",
+				"message": "&cThe First platform will break in 5 seconds!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:124]: "First" would be uncapitalized here.

File: Blitz/Mega Sumo/map.json [hunk 2]
```diff
@@ -151,12 +151,12 @@
 				"commands": []
 			},
 			{
-				"message": "&eThe first platform has crumbled!",
+				"message": "&eThe First platform has crumbled!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:154]: "First" would be uncapitalized here.

File: Blitz/Mega Sumo/map.json [hunk 3]
```diff
@@ -241,42 +241,63 @@
...
-				"interval": 95,
+				"message": "&cThe fourth layer of platforms will break in 4 seconds!",
+				"interval": 86,
 				"repeat": false,
 				"commands": []
 			},
 			{
-				"message": "&cThe World border will start shrinking in 2 seconds!",
-				"interval": 96,
+				"message": "&cThe fourth layer of platform will break in 3 seconds!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:265]: Change "platform" to "platforms"

File: Blitz/Mega Sumo/map.json [hunk 4]
```diff
@@ -241,42 +241,63 @@
...
-				"interval": 97,
+				"message": "&cThe fourth layer of platforms will break in 2 seconds!",
+				"interval": 88,
 				"repeat": false,
 				"commands": []
 			},
 			{
-				"message": "&cOh Boy here we go, &eThe world border is now shrinking!",
-				"interval": 98,
+				"message": "&cThe fourth layer of platform will break in 1 second!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:277]: Change "platform" to "platforms"

File: Blitz/Mega Sumo/map.json [hunk 5]
```diff
@@ -241,42 +241,63 @@
...
+				"commands": []
+			},
+			{
+				"message": "&eThe fourth layer of platforms has crumbled",
+				"interval": 90,
+				"repeat": false,
+				"commands": ["execute at @r run fill 23 7 24 -23 11 -23 air"]
+			},
+			{
+				"message": "&cThe World Border is now shrinking",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:289]: "World Border" -> "world border"

**Review by Owehttamy — CHANGES_REQUESTED** [review]

File: Blitz/Mega Sumo/map.json [hunk 6]
```diff
@@ -241,42 +241,63 @@
...
+				"interval": 100,
+				"repeat": false,
+				"commands": [execute @r run worldborder damage buffer 0]
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:292]: Forgot quotation marks

File: Blitz/Mega Sumo/map.json [hunk 7]
```diff
@@ -115,7 +115,7 @@
         	"limit": 600,
 			"broadcasts": [
 			{
-				"message": "&cWatch out! The first platform will break in 10 seconds!",
+				"message": "&cWatch out! The First platform will break in 10 seconds!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:118]: `				"message": "&cWatch out! The first platform will break in 10 seconds!",`

File: Blitz/Mega Sumo/map.json [hunk 8]
```diff
@@ -127,7 +127,7 @@
 				"commands": []
 			},
 			{
-			    "message": "&cThe First platform will break in 4 seconds!",
+			   	"message": "&cThe First platform will break in 4 seconds!",
```
Comment (Owehttamy) [inline Blitz/Mega Sumo/map.json:130]: `			   	"message": "&cThe first platform will break in 4 seconds!",`

**Review by Owehttamy — CHANGES_REQUESTED** [review]

File: Blitz/Mega Sumo/map.json (hunk 2 above)
Comment (Owehttamy) [reply Blitz/Mega Sumo/map.json:154]: Also still needs to be changed

**Pull request closed without merging**